avocado/core/test.py

```diff
@@ -383,6 +384,11 @@ def _start_logging(self):
                                                        formatter,
                                                        self._ssh_logfile)
 
+        if isinstance(sys.stdout, output.LoggingFile):
+            sys.stdout.add_logger(logging.getLogger("avocado.test.stdout"))
+        if isinstance(sys.stderr, output.LoggingFile):
+            sys.stderr.add_logger(logging.getLogger("avocado.test.stderr"))
+
```

Comment:
^^ btw the travis log comes from modified PR: https://github.com/avocado-framework/avocado/pull/1639